x-pack/plugins/ml/public/application/jobs/jobs_list/components/jobs_list/jobs_list.js

```diff
@@ -202,7 +202,7 @@ export class JobsList extends Component {
         sortable: true,
         truncateText: false,
         dataType: 'number',
-        render: (count) => toLocaleString(count),
+        render: (count) => (count ? toLocaleString(count) : ''),
```

Comment:
this should be an explicit check for `undefined` as a count of `0` would results in and empty string being shown.

Reply:
You're right. I changed it to an explicit check here https://github.com/elastic/kibana/pull/71966/commits/ce1bbc56d06868adc97ca467fb44b6bb898b57eb.